### ISSUE TITLE
BMLPlus doesn't appear when NewPlayer 0.3.3 is installed

### ISSUE THREAD

**Issue by licatglu** [doyaGu/BallanceModLoaderPlus#2]
I'm on Windows 11 and installed Ballance NewPlayer 0.3.3 successfully. I followed all the instructions carefully and checked that BMLPlus.dll is in BuildingBlocks dir and the ModLoader directory is in the root folder of Ballance.

However, upon launching NewPlayer, nowhere does it display BML at the top of the window nor can I find it in the Options menus.

This is a brand new, clean installation on Win 11 with no other older versions previously installed nor any other mods. I've made sure Visual C++ Redistributable for Visual Studio 2015-2022 has been correctly installed on the system.

I tried it on another Win 11 PC and it's also the same.

**Comment by doyaGu** [comment]
Could you please share the Player.log located in the Bin directory, and also check the ModLoader.log in the ModLoader directory?

**Comment by doyaGu** [comment]
I suggest installing or updating your graphics driver; it might resolve the issue.

**Comment by licatglu** [comment]
> Could you please share the Player.log located in the Bin directory, and also check the ModLoader.log in the ModLoader directory?

Hi thanks for replying. I can't find either files in their respective directories. I made sure Show Hidden Files are turned on in File Explorer, but none exists. There is a Player.ini in Bin folder. Is that the one?

Player.ini

`[Startup]
LogMode=1
Verbose=0
ManualSetup=0
LoadAllManagers=1
LoadAllBuildingBlocks=1
LoadAllPlugins=1
[Graphics]
Driver=0
BitsPerPixel=32
Width=800
Height=600
FullScreen=0
DisablePerspectiveCorrection=0
ForceLinearFog=0
ForceSoftware=0
DisableFilter=0
EnsureVertexShader=0
UseIndexBuffers=0
DisableDithering=0
Antialias=0
DisableMipmap=0
DisableSpecular=0
EnableScreenDump=0
EnableDebugMode=0
VertexCache=16
TextureCacheManagement=1
SortTransparentObjects=1
TextureVideoFormat=UNKNOWN_PF
SpriteVideoFormat=UNKNOWN_PF
[Window]
ChildWindowRendering=0
Borderless=0
ClipCursor=0
AlwaysHandleInput=0
PauseOnDeactivated=0
[Game]
Language=1
SkipOpening=0
ApplyHotfix=1
UnlockFramerate=0
UnlockWidescreen=0
UnlockHighResolution=0`

Here are the screenshots for both directories:
![Bin](https://github.com/user-attachments/assets/aea68fb8-c0cc-4e16-9621-6730eba2ccad)
![Modloader](https://github.com/user-attachments/assets/3b25af30-5a99-4c34-bbc5-86102da012fb)

And yes, I uninstalled and re-installed the graphic card driver to the latest version (Intel UHD 31.0.101.1999).

**Comment by doyaGu** [comment]
Please ensure you have the latest x86 version of the Visual C++ Redistributable for Visual Studio 2015-2022 installed. If not, you will need to reinstall it.

**Comment by licatglu** [comment]
> Please ensure you have the latest x86 version of the Visual C++ Redistributable for Visual Studio 2015-2022 installed. If not, you will need to reinstall it.

Yes, I do have the latest installed already. I tried reinstalling it but still the same.

**Comment by doyaGu** [comment]
That's strange. Is your PC an ARM-based system?

**Comment by licatglu** [comment]
> That's strange. Is your PC an ARM-based system?

No, it's i5 10210U laptop with UHD 620 Graphics.

**Comment by doyaGu** [comment]
Ah, I see. You mentioned your graphics card is Intel UHD. Just to confirm, are you using version 0.3.4 of BML+?

**Comment by licatglu** [comment]
Yes I am.
![BMLPlus](https://github.com/user-attachments/assets/a29cc7a3-a787-4291-868e-6ea0dbf77b5e)

**Comment by doyaGu** [comment]
That’s a weird situation, and it’s hard to say what’s causing it. Maybe try waiting for the next release or using another version.

**Comment by licatglu** [comment]
Ok, will do. Thanks for all your help.
